apiman/src/main/java/io/fabric8/apiman/KubernetesServiceCatalog.java

```diff
@@ -79,11 +78,15 @@
 	private List<AvailableServiceBean> searchKube(String keyword){
 		List<AvailableServiceBean> availableServiceBeans = new ArrayList<AvailableServiceBean>();
 		//Obtain a list from Kubernetes, using the Kubernetes API
-		String kubernetesMasterUrl = Systems.getEnvVarOrSystemProperty("KUBERNETES_MASTER", "https://172.28.128.4:8443");
-		String kubernetesNamespace = Systems.getEnvVarOrSystemProperty("KUBERNETES_NAMESPACE", "default");
-		if (Systems.getEnvVarOrSystemProperty(Config.KUBERNETES_TRUST_CERT_SYSTEM_PROPERTY) == null)
-			System.setProperty(Config.KUBERNETES_TRUST_CERT_SYSTEM_PROPERTY, "true");
-		OpenShiftClient osClient = new DefaultOpenShiftClient(kubernetesMasterUrl);
+		KubernetesClient kubernetes = null;
+		String kubernetesMasterUrl = Systems.getEnvVarOrSystemProperty("KUBERNETES_MASTER");
```

Comment:
You probably don't need the `KubernetesClient` instance at all - probably OK to just do `new OpenShiftClient()` & use that for all operations (it extends `KubernetesClient` IIRC)?